### ISSUE TITLE
cellbender failing on sample with ~150 cells

### ISSUE THREAD

**Issue by ktpolanski** [broadinstitute/CellBender#169]
Hello,

Sometimes you're stuck with a sample which doesn't have a lot going on. This is one of those. This is what the knee plot looks like for it:

![image](https://user-images.githubusercontent.com/14993986/204326927-961c9e99-76c9-4d55-b9aa-a315f839cffb.png)

I fed it into a relatively recent cellbender (installed on 08.06.2021, so only a few commits behind master), and got the following strange error about some tensor being empty:

[cbk.5277079.txt](https://github.com/broadinstitute/CellBender/files/10105556/cbk.5277079.txt)

Is there anything that can be done to salvage this sample? Thanks and sorry for the trouble.

**Comment by sjfleming** [comment]
@ktpolanski you found a bug!  Hmmm, I will have to keep that one in mind.  The issue is that I try to compute an "average size" for cells on the first pass through the encoder neural network... but in this sample, by chance, there were actually no "cells" in the first minibatch.  So I will need to fix this.

But for now, I bet there's something you can do.

Try this:

```
cellbender remove-background \
    --input TransTHY8814436/raw.h5 \
    --output TransTHY8814436/TransTHY8814436_knee.h5 \
    --expected-cells 100 \
    --total-droplets-included 1000 \
    --fpr 0.01 0.05 0.1 \
    --cuda
```

(The smaller `--total-droplets-included` will hopefully (1) let CellBender know the prior on empty droplet counts should be larger that it was estimating before, and (2) it will increase the ratio of cells to empties in each minibatch, hopefully side-stepping the bug.)

**Comment by ktpolanski** [comment]
Thanks for the suggestion! Our GPU queues are super congested and the job is still stuck as pending, I'll let you know once it goes through.

**Comment by sjfleming** [comment]
Hopefully these kinds of things are solved in v0.3.0 and much smoother for users.

Closed by #238
